src/domain/post/PostData.ts

```diff
@@ -0,0 +1,36 @@
+
+export default class PostData
+{
+    #id: string;
+    #creatorId: string;
+    #comicId: string;
+
+    #createdAt: Date;
+
+    #ratingCount: number;
+    #reactionCount: number;
+
+    constructor(id: string, creatorId: string, comicId: string, createdAt = new Date(), ratingCount = 0, reactionCount = 0)
+    {
+        this.#id = id;
+        this.#creatorId = creatorId;
+        this.#comicId = comicId;
+
+        this.#createdAt = createdAt;
```

Comment:
The documentation has not been updated with this new name.

Reply:
As discussed, the purpose of the data model is for the initial design only. We've chosen to make the code the source of truth after the design phase to prevent additional overhead.